web/src/utils/tts.tsx

```diff
@@ -0,0 +1,24 @@
+import EasySpeech from 'easy-speech'
+
+const initializedStatus = ['init: failed', 'created']
+
+export const ttsInitialized = (): boolean =>
+  !initializedStatus.some(status => EasySpeech.status().status.includes(status))
+
+const cancelErrorMessages = ['interrupted', 'canceled']
+
+export const isTtsCancelError = (error: unknown): boolean =>
+  error instanceof SpeechSynthesisErrorEvent && cancelErrorMessages.includes(error.error)
+
+const defaultVoices: { [key: string]: string } = {
+  de: 'Anna',
+  en: 'Samantha',
+}
```

Comment:
Could somebody please test these languages on windows/linux? 🙏

Reply:
On linux:
- (firefox) it depends on the third party voices that you use like "Pied" I selected the language and the voice. (I have to reopen my browser to work).
- Chrome works fine.

Reply:
🙃 I would prefer if we had one male and one female default voice. Assistants don't have to be female

Reply:
It was just some voices that worked well across different browsers on MacOS. Feel free to suggest different voices, I am totally fine with any other.

Reply:
I'll give it a try on Monday, when I'm not in a place surrounded by a bunch of people 😁